finality-aleph/Cargo.toml

```diff
@@ -5,8 +5,8 @@ authors = ["Cardinal Cryptography"]
 edition = "2021"
 
 [dependencies]
-aleph-bft = "0.14.0"
-aleph-bft-rmc = "0.3.0"
+aleph-bft = "0.15.0"
+aleph-bft-rmc = "0.4.0"
```

Comment:
It seems to me this was changed to `0.3.1` in the other repo?

Reply:
Let's merge the AlephBFT side and then see if it works :) I'm kind of confused by the `0.3.0` now, does it compile?